internal/mongodoc/doc.go

```diff
@@ -162,6 +162,11 @@ type BaseEntity struct {
 	// Promulgated specifies whether the charm or bundle should be
 	// promulgated.
 	Promulgated IntBool
+
+	// CommonExtraInfo holds arbitrary common extra metadata associated with
+	// the base entity. Thhose data apply to all revisions.
+	// The byte slices hold JSON-encoded data.
+	CommonExtraInfo map[string][]byte `bson:",omitempty" json:",omitempty"`
```

Comment:
This could just be called ExtraInfo as it is on a different document type but is the same thing as the previous one.